syntax = "proto3";

message K8sResources {
  repeated string snippets = 1;
}
message K8sGWRequest {
    int32 user_id = 1;
    int32 max_results = 3;
}
message K8sGWResponse {
    repeated string resources = 1;
    bool status = 2;
}
service K8sGW {
    rpc GetNSs (K8sGWRequest) returns (K8sGWResponse);
    rpc ApplyDeployment (K8sGWRequest) returns (K8sGWResponse);
}